### PR TITLE
[15.0][MIG]hr_expense_cancel

### PR DESCRIPTION
**Pull request by manuelregidor**
Standard migration.

The name of the cancel button has been changed to 'Cancel all related operations', as the hr_expense module in v15 includes a 'Cancel' button that perform less actions than this one.

### CODE REVIEW

**Comment by manuelregidor** [comment]
@HaraldPanten @ValentinVinagre

**Review by ValentinVinagre — APPROVED** [review]
LGTM 👍🏻

**Review by HaraldPanten — CHANGES_REQUESTED** [review]
Hi @manuelregidor 

I think it's not a standard migration, as you've applied some changes in order to adapt the module to make it more compatible with the standard "Cancel" button from V15.

Could you, please:

- Adapt the README file.
- Explain in a comment (here in github, not in the readme file) the changes from V14 to V15for this module. (screenshots are welcome).

After that we can mention the repository PSCs and ask them about their opinion for these changes.

THX!

**Review by HaraldPanten — APPROVED** [review]
Functional review

**Comment by HaraldPanten** [comment]
@pedrobaeza @ivantodorovich Could you have a quick look at this one?

File: hr_expense_cancel/models/hr_expense.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def action_cancel(self):
             if account_move.exists():
                 if account_move.state != "draft":
                     account_move.button_cancel()
-                account_move.with_context({"force_delete": True}).unlink()
+                account_move.with_context(**{"force_delete": True}).unlink()
```
Comment (ivantodorovich) [inline hr_expense_cancel/models/hr_expense.py]: Just use `account_move.with_context(force_delete=True).unlink()`

**Review by ivantodorovich — APPROVED** [review]
LG

File: hr_expense_cancel/models/hr_expense.py (hunk 1 above)
Comment (manuelregidor) [reply hr_expense_cancel/models/hr_expense.py]: Done

**Comment by Saran440** [comment]
Hello All, Should you review  this PR: https://github.com/OCA/hr-expense/pull/56 before migrate to V15?
I found it will conflict with standard odoo.

**Comment by HaraldPanten** [comment]
Hi @Saran440 thx for your comments.

As it's an enhancement, I'd leave that for later (after migration).

For us, having a "gap" in this journal entries sequence (momently) because of cancelling the expenses, hasn't been any problem. As we solve it later.

It's the same when you delete an account move (invoice). You have to consider the sequence as well.

WDYT?

**Comment by HaraldPanten** [comment]
@pedrobaeza Could you merge, please?

THX!

**Comment by pedrobaeza** [comment]
No review by my part

/ocabot migration hr_expense_cancel
/ocabot merge nobump

**Comment by HaraldPanten** [comment]
THX!

**Comment by Saran440** [comment]
@HaraldPanten Sorry for late.
Okay, I will do it later if it helpful with module.

**Comment by HaraldPanten** [comment]
> 

No problem, Just ping me once you make the PR and I'll review 👍

**Pull request merged**